### DISCUSSION TITLE
SEB Server connection over Port 80

### DISCUSSION THREAD

**Discussion by siud112** [SafeExamBrowser/seb-server#80] [Q&A]
Hi,
is it possible to run the SEB server complete over port 80?

Thanks

**Comment by anhefti** [comment]
Hi,

You can but it is insecure and if you want to use proctoring, HTTPS is needed. We recommend to always use HTTPS and configure the nginx proxy that is part of the setup bundle to be the endpoint and map port 443 (default HTTPS port) to the internal HTTP port 8080 of the SEB Server as it is configured per default.

You can change either the Port mapping of the nginx proxy or you can remove the nginx proxy completely and map the default SEB Server port from outside 80 to inside 8080 within the docker-compose.yml 80:8080. You can do that all by editing your  docker-compose.yml file accordingly. Don't forget to adapt also your SEB Server application settings for "sebserver.webservice.http.external.*" to map to SEB Server external URL. 

Best
